interface/forms/patient_encounter/common.php

```diff
@@ -167,9 +167,7 @@ class="css_button link_submit" onClick="top.restoreSession()"><span><?php echo x
  <div class="row">
   <div class="col-xs-12 col-sm-4 col-lg-4 ">
     <?php echo xlt('Consultation Brief Description'); ?>:
-    <textarea class="form-control input-sm" style="resize:none" name='reason' cols='40' rows='12' wrap='virtual'>
-      <?php echo $viewmode ? text($result['reason']) : text($GLOBALS['default_chief_complaint']); ?>
```

Comment:
@aethelwulffe in this case the extra space came from before the `<?php echo ....?>`